### PR TITLE
Jenkinsfile: post-build action which sends a slack notification

### PR DESCRIPTION
**Pull request by ehildenb**
###### Reviewer checklist

- [ ] Test coverage: `stack test --coverage`
- [ ] Public API documentation: `stack haddock`
- [ ] Style conformance: `stylish-haskell`

---

This should enable sending emails on failed jobs to `master` or `[kevm-integration]` tagged commits.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -56,6 +56,13 @@ pipeline {
               '''
             }
           }
+          post {
+            failure {
```
Comment (dwightguth) [inline Jenkinsfile]: I think you probably want regression, not failure here.

Comment (ehildenb) [reply Jenkinsfile]: done, though I think we actually want `failure` because I want it to be annoying if it keeps failing. But we can change it later.

Comment (dwightguth) [reply Jenkinsfile]: Alright. Note that failure will not run if the build is unstable; not sure if you ever get unstable builds, but something to consider.

Comment (ehildenb) [reply Jenkinsfile]: Changed back to `failure`, we can change to `regression` if it becomes annoying.

**Pull request merged**